add.js

```diff
@@ -0,0 +1,207 @@
+const addDescriptionInput = document.getElementById('new-description')
+const addButton = document.getElementById('button')
+const todoListTask = document.getElementById('list-tasks')
+
+/// edits
+
+function editTask(index) {
+
+  const buttonAddTask = document.getElementById('button')
+  const savesTask = document.getElementById(`savesTask${index}`)
+  const titleEdit = document.getElementById(`titleEdit${index}`)
+  const cancelTask = document.getElementById(`cancelTask${index}`)
+  const editTask = document.getElementById(`editTask${index}`)
+  const descriptionEdit = document.getElementById(`descriptionEdit${index}`)
+  const titleNameIndex = document.getElementById(`titleName${index}`)
+  const descriptionNameIndex = document.getElementById(`descriptionName${index}`)
+
+  const listTask = localStorage.getItem("localListTaskObject")
+  const listTaskObject = JSON.parse(listTask)
+
+  buttonAddTask.style.display = "none";
+  savesTask.style.display = "block";
+  savesTask.style.background = "#1E90FF";
+  savesTask.style.color = "white";
+
+  titleNameIndex.style.display = "none";
+  descriptionNameIndex.style.display = "none";
+  editTask.style.display = "none";
+
+  titleEdit.style.display = "block";
+  cancelTask.style.display = "block";
+  descriptionEdit.style.display = "block";
+
+  titleEdit.value = listTaskObject[index].titleName;
+  descriptionEdit.value = listTaskObject[index].descriptionName;
+}
+
+const form = document.getElementById('form');
+const newTitle = document.getElementById('new-title');
+const newDescription = document.getElementById('new-description');
+form.addEventListener('submit', e => {
+  e.preventDefault();
+  checkInputsValue();
+});
+
+function checkInputsValue() {
+
+  const newTitleValue = newTitle.value.trim();
+  const newDescriptionValue = newDescription.value.trim();
+  // Validity
+  if (newTitleValue === '') {
+    setErrorFor(newTitle, 'Bạn Cần Nhập Tiêu Đề Này');
+  } else {
+    setSuccessFor(newTitle);
+  }
+  if (newDescriptionValue === '') {
+    setErrorFor(newDescription, 'Bạn Cần Nhập Tiêu Đề Này');
+  } else {
+    setSuccessFor(newDescription);
+  }
+
+  // set setErrorFor
+  function setErrorFor(input, message) {
+    const formControl = input.parentElement;
+    const p = formControl.querySelector('p');
+    formControl.className = 'form-control error';
+    p.innerText = message;
+  }
+
+  // set setSuccess
+  function setSuccessFor(input) {
+    const formControl = input.parentElement;
+    formControl.className = 'form-control success';
+  }
+  addTasKValue()
+}
+
+// Tao Task
+function addTasKValue() {
+  const newTitleValue = newTitle.value.trim();
+  const newDescriptionValue = newDescription.value.trim();
+  if ((newTitleValue && newDescriptionValue) != 0) {
+    let listTask = localStorage.getItem("localListTaskObject")
+    if (listTask === null) {
+      listTaskObject = [];
+    } else {
+      listTaskObject = JSON.parse(listTask)
+    }
```

Comment:
đoạn này code dài quá, sủ dụng toán tử 3 ngôi đê.
`const listTaskObject = listTask ? JSON.parse(listTask): []`